src/js/map/data-layers.js

```diff
@@ -308,32 +310,67 @@ class DataLayers {
       this.layersData.push({configLayer, layer, data});
     }
     
-    // TODO: add a listener for the map to add/remove the layer based on zoom
-    // this.map.on('zoomend ', (e) => {
-    //   console.log('zoom = '+ this.map.getZoom());
-    //   if ( this.map.getZoom() > 7 ){ this.map.addLayer(layer)}
-    //   else if ( this.map.getZoom() <= 7 ){ this.map.removeLayer(layer)}
-    // });
+    //Listener to control the visibility zoom when zooming
+    this.map.on('zoomend ', (e) => {
+      if(configLayer.displayedFromZoomLevel){
```

Comment:
Shouldn't it be 

if(configLayer.displayedFromZoomLevel or configLayer.displayedUpToZoomLevel)

?